x-pack/legacy/plugins/ml/public/application/services/timefilter_refresh_service.tsx

```diff
@@ -6,4 +6,10 @@
 
 import { Subject } from 'rxjs';
 
-export const mlTimefilterRefresh$ = new Subject();
+interface MlTimeFilterUpdate {
```

Comment:
could this be replaced by `Refresh` from `use_refresh.ts`

Reply:
👍 updated in fada10afc4